ngraph/frontend/tensorflow/src/ngraph_conversions.cpp

```diff
@@ -10,27 +10,25 @@ namespace ov {
 namespace frontend {
 namespace tf {
 
-void NHWCtoNCHW(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& node) {
+void convert_nhwc_to_nchw(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& node) {
     if (need_convert) {
         auto rank = node.get_shape().size();
         if (rank == 4) {
             Transpose<0, 3, 1, 2>(node);
         } else if (rank == 5) {
             Transpose3D<0, 4, 1, 2, 3>(node);
         }
-        SetTracingInfo(op_name, node);
```

Comment:
`set_node_name` is missed. `op_name` is now useless

Reply:
not missed, set_node_name should be called in each translation function separately
![image](https://user-images.githubusercontent.com/21181928/140711672-4dc46cc2-f666-41e7-ad3b-592006db8afc.png)